simulate-event.js

```diff
@@ -272,6 +292,9 @@ module.exports = function (element, type, options) {
     cancelable: true
   }, result(eventOptions, eventType, element, type, options), options);
 
+  // Preserve the original or use the new options if none were given.
+  if (!original) original = options;
```

Comment:
They are blown away in that the new `options` does not have the original parameters.  Yes, this is only when using a previously created event.  I imagine a native Event would not have this problem.

We could have a generic way of handling troublesome members that merit the use of `Object.defineProperty`, but they are currently confined to the deprecated members of `KeyboardEvent`.